universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java

```diff
@@ -94,15 +97,39 @@ public Tag renderFileUploadBlockSubmitForms(
                         renderQuestion(
                             question, rendererParams, applicantQuestionRendererFactory)));
 
+    Tag skipForms = renderDeleteAndContinueFileUploadForms(params);
     Tag buttons = renderFileUploadBottomNavButtons(params);
 
-    return div(formTag, buttons)
+    return div(formTag, skipForms, buttons)
         .withId("azure-upload-form-component")
         .with(
             footer(viewUtils.makeWebJarsTag(WebJarJsPaths.AZURE_STORAGE_BLOB)),
             footer(viewUtils.makeLocalJsTag("azure_upload")));
   }
 
+  @Override
+  protected Optional<ContainerTag> maybeRenderSkipOrDeleteButton(Params params) {
+    if (hasAtLeastOneRequiredQuestion(params)) {
+      // If the file question is required, skip or delete is not allowed.
+      return Optional.empty();
```

Comment:
question (blocking): does the mean if someone uploads the wrong file they'll be unable to delete and upload a different one?

Reply:
This looks like a bug in the AWS implementation, let's not duplicate it here.

Reply:
Thanks! I'm looking into it for AWS, will file a bug if needed.